src/BackgroundLocationTracking/README.md

```diff
@@ -0,0 +1,8 @@
+# Background Location Tracking
+
+BackgroundLocationTracking is a .NET MAUI application designed to track the location of a device in the background using the [ArcGIS Runtime SDK for .NET](https://developers.arcgis.com/net/).
+
+Android application uses Foreground Service to track the location in the background. iOS application uses [SystemLocationDaatSource.AllowBackgroundLocationUpdates](https://developers.arcgis.com/net/api-reference/api/ios/Esri.ArcGISRuntime/Esri.ArcGISRuntime.Location.SystemLocationDataSource.AllowsBackgroundLocationUpdates.html) to track the location in the background.
+
+
+![Screenshot](https://github.com/user-attachments/assets/1d8d251f-5b03-44d0-9d15-dadf54983e70)
```

Comment:
This could use a lot more information, pointing to the bit in iOS that is necessary to set to allow background (info.plist entry + system location datasource properties), and Android wrt foreground service and how it's managed.
Include links to android and apple doc too.